profile/README.md

```diff
@@ -18,13 +16,15 @@ First name Last name [@GitHub username](https://github.com/USERNAME)
 - Hadrien Louis [@hadrylouis](https://github.com/hadrylouis)
 
 ## Students
-
```

Comment:
You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.